torch/autograd/profiler.py

```diff
@@ -1329,6 +1362,11 @@ def append(s):
             row_values.append(evt.node_id)
         if has_input_shapes:
             row_values.append(str(evt.input_shapes)[:shapes_column_width])
+        if with_flops:
+            if evt.flops is None:
+                row_values.append("--")
+            else:
+                row_values.append('{}'.format(evt.flops * (1e-9))[:flops_column_width])
```

Comment:
let's output FLOPs per second - divide by total runtime
also please format, so that there's only e.g. 3-4 digits after the floating point